extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -166,6 +166,7 @@ export const LIFECYCLE_STAGES = {
   layoutAdPromiseDelay: '18',
   signatureVerifySuccess: '19',
   networkError: '20',
+  friendlyIframeIniLoad: '21',
```

Comment:
I discussed this with @keithwrightbos, I believe. Though it could have been @tdrl :)